sidebars.js

```diff
@@ -0,0 +1,49 @@
+// @ts-check
+
+/** @type {import('@docusaurus/plugin-content-docs').SidebarsConfig} */
+const sidebars = {
+  docs: [    {
+      type: 'autogenerated',
```

Comment:
this directs Docusaurus to automatically generate sidebar entries based on the files and folders found in our `docs/` folder. This is how it comes out:
<img width="268" alt="Screen Shot 2022-08-09 at 3 15 14 PM" src="https://user-images.githubusercontent.com/6207644/183742651-7b096937-64fc-466f-8bee-4449b18d14e8.png">